app/routes/armoury/mission-board.tsx

```diff
@@ -1,17 +1,48 @@
 import type { LoaderArgs } from "@remix-run/node"
+import type { z } from "zod"
+import type { MissionBoardSchema } from "~/services/darktide.server"
 import { json } from "@remix-run/node"
 import { useLoaderData } from "@remix-run/react"
 import { reverse, sortBy } from "lodash-es"
 import { getAuthToken } from "~/data/authtoken.server"
 import { authenticator } from "~/services/auth.server"
 import { getMissions } from "~/services/darktide.server"
+import { MissionTimer } from "~/components/MissionTimer"
+import missionLoc from "~/data/exported/mission_localization_en.json"
+import missionInfo from "~/data/exported/mission_info.json"
+import circumstanceInfo from "~/data/exported/circumstance_info.json"
+
+const img_url = (src: string): string => {
+	return `https://darktide-images.vercel.app/_vercel/image?q=100&url=pngs/${src}.png&w=512`
+}
+
+const loc = (key: string): string => {
+	return missionLoc[key as keyof typeof missionLoc] || ""
+}
+
+const determineSecondary = (
+	mission: z.infer<typeof MissionBoardSchema>["missions"][number]
+): string => {
+	if (!mission.extraRewards.sideMission) {
+		return "none"
+	}
+
+	const rewardRatio = mission.extraRewards.sideMission.credits / mission.credits
+	const SCRIPTURE_RATIO = 0.2
+	const GRIMOIRE_RATIO = 0.6
+
+	return rewardRatio > (SCRIPTURE_RATIO + GRIMOIRE_RATIO) / 2
+		? "grimoire"
+		: "scripture"
+}
```

Comment:
Looks like the API returns an optional sideMission property on missions which is either "side_mission_tome" or "side_mission_grimoire"